csv_parser/test_case_generator.py

```diff
@@ -95,8 +161,49 @@ def run(perimeters):
     return test_cases
 
 
+def generate_step_json(perimeter, test_case, row, receive_json):
+    # Create the JSON file for the step
+    if not os.path.exists(f'./out/test-cases/{perimeter["name"]}/{test_case["label"]}'):
+        os.makedirs(f'./out/test-cases/{perimeter["name"]}/{test_case["label"]}')
+    with open(
+            f'./out/test-cases/{perimeter["name"]}/{test_case["label"]}/{len(test_case["steps"])}-{row["Pas de test"]} {row["Modèle"]}.json',
+            'w', encoding='utf-8') as file:
+        file.write(dumps(receive_json, indent=4, ensure_ascii=False))
+
+
 def get_type(type_in_french):
     if type_in_french == "Envoi":
         return "send"
     elif type_in_french == "Réception":
         return "receive"
+
+
+def generate_type_override_map(row):
+    # We load the relevant schema from ../src/main/resources/json-schema/row["Pas de test"].schema.json into the
+    # schema object
+    schema = json.load(open(f'../src/main/resources/json-schema/{row["Modèle"]}.schema.json'))
+    # We iterate over all properties recursively in the schem and add those that have simple types different from
+    # string (i.e. integer, number, boolean) to the type_override_map object
+    type_override_map = {}
+    for key, value in nested_dict_iter(schema["definitions"]):
+        if "type" in value and not isinstance(value["type"], collections.Mapping) and value["type"] not in {"string",
```

Comment:
C'est possible d'avoir `"type" in value` à false ?
Ca sert à quoi : `not isinstance(value["type"], collections.Mapping)` ?

Reply:
C'est possible, parce qu'on va parcourir les proprietes de chaque definition, ex: 
![image](https://github.com/ansforge/SAMU-Hub-Modeles/assets/105354733/d000596b-a993-4dcb-9338-fa2055ff5554)

Ca sert a gerer un cas limite present dans l'objet 'decision' qui a une propriete qui s'appelle 'type', qui est donc traité par pandas comme un dict: 
![image](https://github.com/ansforge/SAMU-Hub-Modeles/assets/105354733/b19363f8-04a8-4b4f-8553-a55aa613ee1d)

Reply:
arg ok... après, on peut aussi bloquer certains types de champs "de base" dans le MDD un peu comme pour file https://github.com/ansforge/SAMU-Hub-Modeles/blob/5eb74b453bbfeb195ba7b59a0274d1b3adeca44b/csv_parser/csv_parser.py#L264-L270 pour éviter les soucis si besoi